meta-luneos/recipes-core/packagegroups/packagegroup-webos-extended.bb

```diff
@@ -238,6 +238,9 @@ WEBOS_FOSS_MISSING_FROM_RDEPENDS = " \
     sqlite3 \
 "
 
+RDEPENDS:${PN} += "${MACHINE_EXTRA_RDEPENDS}"
+RRECOMMENDS:${PN} += "${MACHINE_EXTRA_RRECOMMENDS}"
```

Comment:
I'm a bit surprised Yocto doesn't already take care of this stuff. @shr-project what's your opinion?

Reply:
packagegroup-machine-base in oe-core takes care of this, but our images don't use this packagegroup.

there is also
meta-webos-raspberrypi/conf/machine/include/webos-rpi.inc:MACHINE_EXTRA_RRECOMMENDS:remove = "kernel-modules"
meta-webos-raspberrypi/conf/machine/include/webos-rpi.inc:MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS:remove = "kernel-modules"

in OSE to avoid pulling all of the enabled kernel modules into the image (and install only the ones we really need).